modules/social_features/social_group/modules/social_group_secret/social_group_secret.info.yml

```diff
@@ -0,0 +1,8 @@
+name: 'Social Group Secret'
+description: 'Provides Secret Group feature.'
+type: module
+core: 8.x
+dependencies:
+  - r4032login:r4032login
```

Comment:
Is there a reason we depend on the r4032login module? I don't think that's right

Reply:
@Kingdutch it's needed because secret groups module extending event subscriber of this module.